extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php

```diff
@@ -303,15 +305,4 @@ function getDefaultQuery() {
 		$query = parent::getDefaultQuery();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$query". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)